src/_premake_init.lua

```diff
@@ -468,6 +468,7 @@
 		kind  = "list:string",
 		allowed = {
 			"Component",           -- DEPRECATED
+			"ConformanceMode",
```

Comment:
We don't add to `flags` anymore, you should create a new API called `conformancemode` with `kind = boolean`.